pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -1492,6 +1537,26 @@ func (f *schemaChangeFrontier) SpanFrontier() *span.Frontier {
 	return f.spanFrontier.Frontier
 }
 
+func (f *schemaChangeFrontier) checkpoint(c *jobspb.ChangefeedProgress_Checkpoint) {
```

Comment:
nit: naming: Might be nice to name this something getCheckpoint or populateCheckpoint to make it clear that c is an out param from the call site.